modules/data/navigation.components.json

```diff
@@ -146,5 +146,17 @@
         "groups": [
             "doNotCache"
         ]
+    },
+
+    "accept-cookies": {
+    "selectorData": "L2AGLb",
```

Comment:
Generally, we don't want to select on ids or classes that look like this--often a base64 number or hash. These are generated by various web frameworks and can vary from version to version of the website, or even by the session. I suspect this is why it is failing in CI.

Reply:
Good point. Hmm I can't find a better strategy for that element other than it's visible text.

Can we change the strategy to xpath and the selectorData to "//div[text()='Accept all']" ? It should work.

Reply:
I just made a commit with all the requested changes.